File: Assets/VirtualCity/ProtoDefine/ReqGetProxyMessage.cs
using System.Collections.Generic;
using ProtoBuf;
namespace ProtoDefine {
[ProtoContract]


public class ReqGetProxyMessage {
    //推荐码
[ProtoMember(1)]
    public string code;
    //0:支付宝
    //1:微信
[ProtoMember(2)]
    public int payType;

    public int getPayType() {
        return payType;
    }

    public void setPayType(int payType) {
        this.payType = payType;
    }

    public ReqGetProxyMessage() {
    }

    public string getCode() {
        return code;
    }

    public void setCode(string code) {
        this.code = code;
    }

    public ReqGetProxyMessage(string code) {
        this.code = code;
    }
}
}

File: Assets/VirtualCity/ProtoDefine/Java/system/ReqGetProxyMessage.java
package com.kingston.jforgame.server.game.system.message;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import com.kingston.jforgame.server.game.Modules;
import com.kingston.jforgame.server.game.system.SystemDataPool;
import com.kingston.jforgame.socket.annotation.MessageMeta;
import com.kingston.jforgame.socket.message.Message;

@MessageMeta(module = Modules.SYSTEM,cmd = SystemDataPool.REQ_GETPORXY)
public class ReqGetProxyMessage extends Message {
    //推荐码
    @Protobuf(order = 1)
    private String code;
    //0:支付宝
    //1:微信
    @Protobuf(order = 2)
    private int payType;

    public int getPayType() {
        return payType;
    }

    public void setPayType(int payType) {
        this.payType = payType;
    }

    public ReqGetProxyMessage() {
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public ReqGetProxyMessage(String code) {
        this.code = code;
    }
}
